apps/core/templates/core/home-authenticated.html

```diff
@@ -3,32 +3,214 @@
 {% load i18n %}
 
 {% block content %}
-  <h2>{% trans "Welcome to Pixel" %}</h2>
+<style> html{min-width: 600px} </style>
+<h2 style="text-align : center;">{% trans "Welcome to Pixel" %}</h2>
 
+<div style="text-align:center; width:100%; min-width:500px">
+    <a
+      href="{% url 'explorer:pixelset_list' %}"
+      class="button"
+    >
+      {% trans "Search in Pixel" %}
+    </a>
+</div>
+
+<h4 style="text-align: center;"> Different organisms... </h4>
+  <div class="grid-x">
```

Comment:
wrong indentation. I think you should divide the page with the Foundation grid: https://foundation.zurb.com/sites/docs/xy-grid.html.

1 row for each "section", including the title on its own (sub-)row and two columns for the graph and the content on the right. You can use Foundation CSS classes directly here but usually we use SCSS mixins in the `.scss` files and use meaningful CSS class names. But it is fine if you do use the Foundation classes here.